### ISSUE TITLE
Introduction For My LossFunction

### ISSUE THREAD

**Issue by hi-sushanta** [pytorch/pytorch#104706]
### 🚀 The feature, motivation, and pitch

It's used most of the time in face detection and object detection project because this loss function's main job is to calculate the difference between the ground truth bounding box and the predicted bounding box. And also this loss function is called Localization Loss.

First please looked this image : <img  src="https://i.stack.imgur.com/nQuCg.jpg"  alt="Formula"/>

You only looked above image's first and second formulas similar to my OBJLoss without using a square root. 

And if you don't sure what area to use so please looked this notebook: https://drive.google.com/file/d/1Hpe4jx_ZIlwW4C9f5A8AMddI62Mee6-M/view?usp=sharing
